metals/src/main/scala/scala/meta/internal/metals/codeactions/StringToMultiline.scala

```diff
@@ -0,0 +1,66 @@
+package scala.meta.internal.metals.codeactions
+
+import scala.meta.internal.metals.CodeAction
+import scala.meta.internal.metals.MetalsEnrichments._
+import org.eclipse.{lsp4j => l}
+import scala.concurrent.{ExecutionContext, Future}
+import scala.meta.pc.CancelToken
+import scala.meta.internal.metals.Compilers
+import scala.meta.tokens.Token
+
+class StringToMultiline(compilers: Compilers) extends CodeAction {
+
+  override def kind: String = l.CodeActionKind.Refactor
+
+  override def contribute(
+      params: l.CodeActionParams,
+      token: CancelToken
+  )(implicit ec: ExecutionContext): Future[Seq[l.CodeAction]] = {
+
+    val uri = params.getTextDocument.getUri
+    val path = uri.toAbsolutePath
+    val position = params.getRange.getStart
+
+    Future
+      .successful {
+        path.toInput.tokenize.toOption match {
```

Comment:
You should use the content from the buffers here. `path.toInput` will pick up the filesystem version. We keep the most recent version of the file in the `Buffers`.

Reply:
okay perfect I'll try that :)